src/include/daos/mem.h

```diff
@@ -100,7 +100,7 @@ struct umem_store_ops {
 	int	(*so_flush_prep)(struct umem_store *store, struct umem_store_iod *iod,
 				 daos_handle_t *fh);
 	int	(*so_flush_copy)(daos_handle_t fh, d_sg_list_t *sgl);
-	int	(*so_flush_post)(daos_handle_t fh, int err);
+	int (*so_flush_post)(daos_handle_t fh, int err);
```

Comment:
unexpected change?

Reply:
I have clang-format enabled so it probably did this.